tests/integration/test_fim/test_files/test_max_eps/test_max_eps.py

```diff
@@ -171,7 +171,7 @@ def test_max_eps(configure_local_internal_options_module, get_configuration, con
                                      error_message=ERR_MSG_MONITORING_PATH).result()
     create_multiple_files(get_configuration)
     # Create files to read max_eps files with added events
-    n_results = max_eps + 5
+    n_results = max_eps * 2
```

Comment:
Could you explain the reason to change by `*2`, please?

Reply:
This was changed because the max_eps+5 was the number of files that would be created, and then the number of results the FileMonitor is trying to get. The main issue was that this that later the results are filtered and it is checked if the number that  happened a specific second matches the max_eps. having just 5 additional events is too small of a margin because the first/last 6 events could happen in another second (before or after) and that caused the event test to fail. By doubling the ammount of files/events generated, Now the having less than the expected events does not happen.